onnxruntime/core/framework/session_state.cc

```diff
@@ -186,7 +186,7 @@ Status SessionState::CreateKernels(const KernelRegistryManager& kernel_registry_
 const SequentialExecutionPlan* SessionState::GetExecutionPlan() const { return p_seq_exec_plan_.get(); }
 
 Status SessionState::AddInitializedTensor(int ort_value_index, const OrtValue& ort_value, const OrtCallback* d,
-                                          bool constant) {
+                                          bool constant, bool sparse) {
```

Comment:
Is there a way to ask OrtValue if the underlying tensor is sparse than having a separate boolean?

Reply:
Currently, there is not a way. This is because initializers are still stored internally as Tensors. It is thought that eventually, we will always store sparse as SparseTensors. However, currently all the ops take Tensors so not a reason to change it now, but sometimes we need to know if it was originally sparse.